CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Changes
+
+* [#406](https://github.com/rubocop-hq/rubocop-rails/pull/406): Deconstruct "table.column" in Rails/WhereEquals. ([@mobilutz][])
```

Comment:
Can you rebase with the latest master branch and enclose the cop name in backquotes?

```suggestion
* [#406](https://github.com/rubocop-hq/rubocop-rails/pull/406): Deconstruct "table.column" in `Rails/WhereEquals`. ([@mobilutz][])
```

Reply:
Done.